examples/producer_consumer.py

```diff
@@ -22,16 +22,18 @@ def main():
     def produce():
         for i in range(10):
             msgs.send(i)
-        done.send()
+        msgs.close()
 
     def consume(name):
         for msg in msgs:
             out.send('%s:%s ' % (name, msg))
+        out.close()
 
     def logger():
         for msg in out:
             sys.stdout.write(msg)
```

Comment:
The following line was not getting run before because the call would hang inside `out._next()` when doing a blocking receive on a channel that was never closed.